src/main/resources/assets/additionaladditions/lang/vi_vn.json

```diff
@@ -0,0 +1,88 @@
+  "item.additionaladditions.music_disc_1007.desc": "adoghr - 1007",
+  "item.additionaladditions.music_disc_1507.desc": "adoghr - 1507",
+  "item.additionaladditions.mysterious_bundle": "Túi bọc bí ẩn",
+  "item.additionaladditions.pocket_jukebox": "Máy phát nhạc bỏ túi",
+  "item.additionaladditions.chicken_nugget": "Chicken Nugget",
+  "item.additionaladditions.gold_ring": "Nhẫn vàng",
+  "block.additionaladditions.glow_stick": "Gậy phát sáng",
+  "block.additionaladditions.copper_patina": "Lớp gỉ đồng",
+  "block.additionaladditions.rope": "Dây thừng",
+  "block.additionaladditions.amethyst_lamp": "Đèn thạch anh tím",
+  "block.additionaladditions.patina_block": "Khối lớp gỉ",
+  "enchantment.additionaladditions.precision": "Độ chính xác",
+  "enchantment.additionaladditions.precision.desc": "Cải thiện độ chính xác của nỏ.",
+  "enchantment.additionaladditions.speed": "Tốc độ",
+  "enchantment.additionaladditions.speed.desc": "Tăng tốc độ di chuyển.", 
+  "entity.additionaladditions.glow_stick": "Gậy phát sáng",
+
+  "depth_meter.elevation": "Độ cao: %s",
+
+  "item.minecraft.potion.effect.haste_potion": "Thuốc của sự vội vàng",
+  "item.minecraft.potion.effect.long_haste_potion": "Thuốc của sự vội vàng",
+  "item.minecraft.potion.effect.strong_haste_potion": "Thuốc của sự vội vàng",
+  "item.minecraft.splash_potion.effect.haste_potion": "Thuốc của sự vội vàng có thể ném được",
+  "item.minecraft.splash_potion.effect.long_haste_potion": "Thuốc của sự vội vàng có thể ném được",
+  "item.minecraft.splash_potion.effect.strong_haste_potion": "Thuốc của sự vội vàng có thể ném được",
+  "item.minecraft.lingering_potion.effect.haste_potion": "Thuốc của sự vội vàng kéo dài",
+  "item.minecraft.lingering_potion.effect.long_haste_potion": "Thuốc của sự vội vàng kéo dài",
+  "item.minecraft.lingering_potion.effect.strong_haste_potion": "Thuốc của sự vội vàng kéo dài",
+  "item.minecraft.tipped_arrow.effect.haste_potion": "Mũi tên vội vàng",
+  "item.minecraft.tipped_arrow.effect.long_haste_potion": "Mũi tên vội vàng",
+  "item.minecraft.tipped_arrow.effect.strong_haste_potion": "Mũi tên vội vàng",
+
+  "additionaladditions.advancements.obtain_adoghr_disc": "adoghr",
+  "additionaladditions.advancements.obtain_adoghr_disc.desc": "Sở hữu một trong những đĩa nhạc của adoghr",
+  "additionaladditions.advancements.obtain_chicken_nugget": "What a Nugget, Once Again!",
+  "additionaladditions.advancements.obtain_chicken_nugget.desc": "Find a Chicken Nugget in the wilderness",
+  "additionaladditions.advancements.obtain_gilded_netherite": "Mạ vàng",
+  "additionaladditions.advancements.obtain_gilded_netherite.desc": "Sở hữu áo giáp Netherit mạ vàng",
+  "additionaladditions.advancements.obtain_glow_stick": "Sáng lên đi em ei!",
+  "additionaladditions.advancements.obtain_glow_stick.desc": "Dùng Gậy phát sáng để thắp sáng hang động",
+  "additionaladditions.advancements.obtain_mysterious_bundle": "Nhắm mắt mua liều",
```

Comment:
Inappropriate? Translates to "Close your eyes and buy dose" when using Google translate

Reply:
According to wikipedia, this string should be something like "mua mèo trong bị" instead

Reply:
Nhắm mắt mua liều has the same meaning, we can use any of them
Here is the source for nhắm mắt mua liều: https://vtudien.com/anh-viet/dictionary/nghia-cua-tu-buy%20a%20pig%20in%20a%20poke%20(to%20...)

Reply:
> Inappropriate? Translates to "Close your eyes and buy dose" when using Google translate

it's a proverb, google can't really handle Vietnamese (especially proverb) because we can switch the word's position to have another sentence that has an entirely different meaning